conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -71,6 +72,13 @@ class GramineEnclaveHandle(
         return process.inputStream.reader().use { it.readText() }.trimEnd()
     }
 
+    // ProcessBuilder.Redirect.DISCARD
+    // was introduce in Java 9 but Enclave still supports Java 8.
+    // That is why NULL_FILE was added in. The code was copied from ProcessBuilder.Redirect.DISCARD
+    private val NULL_FILE: File = File(
```

Comment:
Can this go in the companion object? Also, is there a missing space before "NUL"?